src/main/java/org/opensearch/security/rest/SecurityInfoAction.java

```diff
@@ -79,9 +78,13 @@ public SecurityInfoAction(final Settings settings, final RestController controll
         this.evaluator = evaluator;
     }
 
-    @Override
     public List<Route> routes() {
```

Comment:
same

Reply:
My bad, they should be deleted as well, fixed